### PR TITLE
Add lemma prob_div_mul_relf, refactor code in proofs of BAYES_THEOREM, and COND_PROB_ADDITIVE

### PR DESCRIPTION
**Pull request by KaiPhan**

### CODE REVIEW

**Comment by binghe** [comment]
@KaiPhan HOL's core theories (proof scripts under `src/`) cannot contain Unicode characters (with a few exceptions like term quotations). Please replace the Unicode characters (`∀`, `≠`, ..) from your changes. In Emacs, this can be done by the menu item _HOL--Misc--Clean up (...)_

**Comment by mn200** [comment]
As _per_ the regression test machinery, you have used "illegal" Unicode characters under `src/`.  See details at

  https://github.com/HOL-Theorem-Prover/HOL/blob/develop/Manual/Developers/developers.md#coding-standardsrequirements

The emacs-mode has a menu entry (bottom of HOL/Misc) that removes common Unicode and replaces it with the correct ASCII equivalent.

**Comment by KaiPhan** [comment]
@binghe @mn200 Thanks for reviewing. I already updated code. Could you please take a look?

**Comment by binghe** [comment]
> @binghe @mn200 Thanks for reviewing. I already updated code. Could you please take a look?

Yes, new tests have passed. Thanks.

File: src/probability/probabilityScript.sml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+(* -*- Mode: Holscript; -*- *)
```
Comment (mn200) [inline src/probability/probabilityScript.sml:1]: Please delete this line; if you're using emacs, you should set up your editor configuration to automatically recognise `*Script.sml` as files that should use `holscript-mode`.

Comment (KaiPhan) [reply src/probability/probabilityScript.sml]: @mn200 I got this. And already updated code.

**Comment by mn200** [comment]
Thanks!  This enables nice simplifications of proofs.

**Pull request merged**